Makefile

```diff
@@ -19,33 +19,33 @@ build:
-		--import-mappings SecurityContext=io.kubernetes.client.models.V1SecurityContext \
-		--import-mappings Toleration=io.kubernetes.client.models.V1Toleration \
-		--import-mappings Volume=io.kubernetes.client.models.V1Volume \
-		--import-mappings VolumeDevice=io.kubernetes.client.models.V1VolumeDevice \
-		--import-mappings VolumeMount=io.kubernetes.client.models.V1VolumeMount \
+		--import-mappings Affinity=io.kubernetes.client.openapi.models.V1Affinity \
+		--import-mappings ConfigMapKeySelector=io.kubernetes.client.openapi.models.V1ConfigMapKeySelector \
+		--import-mappings Container=io.kubernetes.client.openapi.models.V1Container \
+		--import-mappings ContainerPort=io.kubernetes.client.openapi.models.V1ContainerPort \
+		--import-mappings EnvFromSource=io.kubernetes.client.openapi.models.V1EnvFromSource \
+		--import-mappings EnvVar=io.kubernetes.client.openapi.models.V1EnvVar \
+		--import-mappings HostAlias=io.kubernetes.client.openapi.models.V1HostAlias \
+		--import-mappings Lifecycle=io.kubernetes.client.openapi.models.V1Lifecycle \
+		--import-mappings ListMeta=io.kubernetes.client.openapi.models.V1ListMeta \
+		--import-mappings LocalObjectReference=io.kubernetes.client.openapi.models.V1LocalObjectReference \
+		--import-mappings ObjectMeta=io.kubernetes.client.openapi.models.V1ObjectMeta \
+		--import-mappings ObjectReference=io.kubernetes.client.openapi.models.V1ObjectReference \
+		--import-mappings PersistentVolumeClaim=io.kubernetes.client.openapi.models.V1PersistentVolumeClaim \
+		--import-mappings PodDisruptionBudgetSpec=io.kubernetes.client.openapi.models.V1beta1PodDisruptionBudgetSpec \
+		--import-mappings PodDNSConfig=io.kubernetes.client.openapi.models.V1PodDNSConfig \
+		--import-mappings PodSecurityContext=io.kubernetes.client.openapi.models.V1PodSecurityContext \
+		--import-mappings Probe=io.kubernetes.client.openapi.models.V1Probe \
+		--import-mappings ResourceRequirements=io.kubernetes.client.openapi.models.V1ResourceRequirements \
+		--import-mappings SecretKeySelector=io.kubernetes.client.openapi.models.V1SecretKeySelector \
+		--import-mappings SecurityContext=io.kubernetes.client.openapi.models.V1SecurityContext \
+		--import-mappings Toleration=io.kubernetes.client.openapi.models.V1Toleration \
+		--import-mappings Volume=io.kubernetes.client.openapi.models.V1Volume \
+		--import-mappings VolumeDevice=io.kubernetes.client.openapi.models.V1VolumeDevice \
+		--import-mappings VolumeMount=io.kubernetes.client.openapi.models.V1VolumeMount \
 		--generate-alias-as-model
 	# add the io.kubernetes:java-client to the deps
-	sed 's/<dependencies>/<dependencies><dependency><groupId>io.kubernetes<\/groupId><artifactId>client-java<\/artifactId><version>5.0.0<\/version><\/dependency>/g' pom.xml > tmp && mv tmp pom.xml
+	sed 's/<dependencies>/<dependencies><dependency><groupId>io.kubernetes<\/groupId><artifactId>client-java<\/artifactId><version>9.0.0<\/version><\/dependency>/g' pom.xml > tmp && mv tmp pom.xml
```

Comment:
If too soon for `10.0.0` which came out a few days ago, can we bump to `9.0.2` to get the bugfixes?

Reply:
Makes sense.